composer.json

```
@@ -68,7 +68,7 @@
         "symfony/maker-bundle": "To use the command for autogenerating configuration"
     },
     "conflict": {
-        "phpoffice/phpspreadsheet": ">=2"
+        "phpoffice/phpspreadsheet": "<1.23.0"
```

I think this is better, right?

```suggestion
        "phpoffice/phpspreadsheet": "<2"
```

---

Ah, correct, they added the new API in that version!